addons/io_hubs_addon/components/types.py

```diff
@@ -23,6 +23,7 @@ class Category(Enum):
     MISC = 'Misc'
     LIGHTS = 'Lights'
     MEDIA = 'Media'
+    USER = 'User'
```

Comment:
The new `USER` type doesn't seem to be used anywhere currently.  Is it just another option, or was it supposed to be automatically assigned to user definitions somehow?

Reply:
It's only used in external components like here: https://github.com/Hubs-Foundation/hubs-postprocessing-addon/blob/main/blender/post_processing_effects.py